docker/volumeplugin/rancher.go

```diff
@@ -50,7 +50,7 @@ func NewRancherState(driver string, client *client.RancherClient) (*RancherState
 }
 
 func (r *RancherState) IsCreated(name string) (bool, error) {
-	_, vol, err := r.Get(name)
+	_, vol, err := r.GetAny(name)
```

Comment:
IsCreated shouldn't change because this method is used to determine if the volume has physically been created.  A volume is only created if in the goodStates.  Did you see some reason to change this method?